src/components/TrackTile/TrackerDetails/TrackerHistoryChart/Bar.tsx

```diff
@@ -35,34 +36,38 @@ const Bar: FC<BarProps> = (props) => {
   }, [animated, percentComplete]);
 
   return (
-    <Animated.View
-      testID={testID}
-      style={[
-        {
-          height: ref.current.interpolate({
-            inputRange: [0, 1],
-            outputRange: ['-0.1%', '100%'],
-          }),
-        },
-        barStyle,
-        style,
-        {
-          backgroundColor: color || styles.backgroundColor?.backgroundColor,
-        },
-      ]}
-    />
+    <View style={containerStyle}>
+      <Animated.View
+        testID={testID}
+        style={[
+          {
+            width: ref.current.interpolate({
+              inputRange: [0, 1],
+              outputRange: ['0%', '100%'],
```

Comment:
You might double check this but I think I had this as `-0.1%` because the animation looked better when decreasing a value to 0. I think at 0% it just kind of disappears but the negative value has it shrink down correctly.

Reply:
I was experiencing a problem with an empty graph rendering as full with the negative value only when the bar is horizontal.

Reply:
Gotcha 👍 yeah this always seemed like a hack anyways.